<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OurMainWindow</class>
 <widget class="QMainWindow" name="OurMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1512</width>
    <height>1134</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1512</width>
    <height>1134</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1512</width>
    <height>1134</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OurMainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#OurMainWindow{background-image: url(:/new/prefix1/picture/background_new.png);}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>610</y>
      <width>408</width>
      <height>135</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>406</width>
      <height>134</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/picture/start_new.png);
border:1px solid rgba(255,255,255,0);
padding:0px;
pressed{border:1px solid rgba(255,255,255,0);
padding:0px;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>820</y>
      <width>240</width>
      <height>75</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>75</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>75</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/picture/record_new_new.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>950</y>
      <width>240</width>
      <height>75</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>75</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>75</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/picture/gameover_new.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>9</y>
      <width>32</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/picture/help.png);
border:1px solid rgba(255,255,255,0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1444</x>
      <y>7</y>
      <width>45</width>
      <height>38</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>38</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>45</width>
      <height>38</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/picture/music_on.png);
border:1px solid rgba(255,255,255,0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>38</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1445</x>
      <y>7</y>
      <width>24</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/picture/music_off.png);
border:1px solid rgba(255,255,255,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1512</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
